### PR TITLE
Functional tests - Enable disable accented url

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Enable disable accented url
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/13_shopParameters/06_trafficAndSeo/seoAndUrls/04_enableDisableAccentedURL" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17375)

### CODE REVIEW

File: tests/puppeteer/campaigns/functional/BO/13_shopParameters/06_trafficAndSeo/seoAndUrls/04_enableDisableAccentedURL.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    {args: {action: 'disable', enable: false, productNameInURL: productNameWithoutAccent}},
+  ];
+
+  tests.forEach((test) => {
+    it('should go to \'Shop parameters > SEO and Urls\' page', async function () {
+      await this.pageObjects.boBasePage.goToSubMenu(
+        this.pageObjects.boBasePage.shopParametersParentLink,
+        this.pageObjects.boBasePage.trafficAndSeoLink,
+      );
+      await this.pageObjects.boBasePage.closeSfToolBar();
```
Comment (boubkerbribri) [inline tests/puppeteer/campaigns/functional/BO/13_shopParameters/06_trafficAndSeo/seoAndUrls/04_enableDisableAccentedURL.js]: no need for this already closed in line 55

File: tests/puppeteer/campaigns/functional/BO/13_shopParameters/06_trafficAndSeo/seoAndUrls/04_enableDisableAccentedURL.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      const result = await this.pageObjects.seoAndUrlsPage.enableDisableAccentedURL(test.args.enable);
+      await expect(result).to.contains(this.pageObjects.seoAndUrlsPage.successfulSettingsUpdateMessage);
+    });
+
+    it('should go to \'Catalog > Products\' page', async function () {
+      await this.pageObjects.boBasePage.goToSubMenu(
+        this.pageObjects.boBasePage.catalogParentLink,
+        this.pageObjects.boBasePage.productsLink,
+      );
+      await this.pageObjects.boBasePage.closeSfToolBar();
```
Comment (boubkerbribri) [inline tests/puppeteer/campaigns/functional/BO/13_shopParameters/06_trafficAndSeo/seoAndUrls/04_enableDisableAccentedURL.js]: same here

**Review by boubkerbribri — COMMENTED** [review]
Thanks @nesrineabdmouleh

**Review by matthieu-rolland — APPROVED** [review]
lgtm

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**